lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -491,7 +521,8 @@ def create_volume(self):
             self.create_volume_async()
         else:
             options = self.create_volume_options()
-            volume_create = netapp_utils.zapi.NaElement.create_node_with_children('volume-create', **options)
+            volume_create = netapp_utils.zapi.NaElement.create_node_with_children('volume-create',
```

Comment:
I don't object to the reformatting, but is it really necessary?

Are you using some formatting tool?

Reply:
reverting